autoscalers.go

```diff
@@ -0,0 +1,99 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type AutoscalersService interface {
+	AutoscalersList(app string) ([]Autoscaler, error)
+	AutoscalerAdd(app string, params AutoscalerAddParams) (*Autoscaler, error)
+	AutoscalerRemove(app string, id string) error
+}
+
+var _ AutoscalersService = (*Client)(nil)
+
+type Autoscaler struct {
+	ID            string  `json:"id"`
+	ContainerType string  `json:"container_type"`
+	Metric        string  `json:"metric"`
```

Comment:
Would be nice to habe a list of all the available metrics here.

Reply:
I don't really get what you mean here

Reply:
Would be nice if you could add constants with all the possible metrics.

Reply:
We actually have an endpoint in the alerter which lists all the metrics (cf. Scalingo/scalingo-alerter-service#54)